src/backend/main/py.main/caliopen_main/objects/message.py

```diff
@@ -127,8 +128,12 @@ def create_draft(cls, user_id=None, **params):
 
     def patch_draft(self, patch, **options):
         """Operation specific to draft, before applying generic patch."""
-        self.get_db()
-        self.unmarshall_db()
+        try:
+            self.get_db()
+            self.unmarshall_db()
+        except:
```

Comment:
it could be many different exception than a NotFound (connection error, etc)

It's not valid to catch all exception into such NotFound